wix/windows-icu-msm.wxs

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <Module Id="dt.compnerd.org.SwiftRuntime.ICU" Language="1033" Version="64.1">
```

Comment:
Is there a reason to put this into the `SwiftRuntime` namespace rather than having a top-level namespace?  Id be fine with `SwiftRuntimeICU` to differentiate it from a standard ICU build.